app/Services/Manager.php

```diff
@@ -198,6 +198,25 @@ public function collectContestInfo($data)
     }
 
     /**
+     * Get a collection of northstar users that exist on a model.
+     *
+     * @param  \Gladiator\Models\Competition|WaitingRoom $model
+     *
+     * @return collection $users
+     */
+    public function getModelUsers($model)
+    {
+        $users = [];
+        $ids = $model->users->pluck('id')->toArray();
+
+        if ($ids) {
+            $users = $this->userRepository->getAll($ids);
+        }
+
+        return $users;
```

Comment:
to maybe clean this up in case `$ids` is null or a model is passed that doesn't have users...

```php
if ($ids) {
  return $this->userRepository->getAll($ids);
}

return null;
```